runtime/queries/metricsview.go

```diff
@@ -404,3 +409,107 @@ func writeXLSX(meta []*runtimev1.MetricsViewColumn, data []*structpb.Struct, wri
 
 	return err
 }
+
+func writeParquet(meta []*runtimev1.MetricsViewColumn, data []*structpb.Struct, ioWriter io.Writer) error {
+	fields := make([]arrow.Field, 0, len(meta))
+	for _, f := range meta {
+		arrowField := arrow.Field{}
+		arrowField.Name = f.Name
+		switch f.Type {
+		case "CODE_BOOL":
```

Comment:
Can use the generated constants instead, like `runtimev1.Type_CODE_BOOL`

Reply:
we can have it like this:
```go
		typeCode := runtimev1.Type_Code(runtimev1.Type_Code_value[f.Type])
		switch typeCode {
		case runtimev1.Type_CODE_BOOL:
			arrowField.Type = arrow.FixedWidthTypes.Boolean
```